clang/test/AST/mangle-lambda-expression-no-crash.cpp

```diff
@@ -0,0 +1,4 @@
+// RUN: %clang_cc1 -fsyntax-only -std=c++20 -verify %s
```

Comment:
`-fsyntax-only` is suspicious, i would prefer a CodeGenTest that checks the mangle name
Maybe moving the test to `test/CodeGenCXX/mangle-lambdas.cpp ` (or one of the other similar test in this folder) would be a good idea

Reply:
Sorry, I am afraid that I misunderstand root cause of this issue and still need more working on it.